types/fontname/index.d.ts

```diff
@@ -0,0 +1,14 @@
+interface FontNameData {
+    postScriptName: string;
+    family: string;
+    style: string;
+    fullName: string;
+    version: string;
+}
+
+interface FontName {
+    parse(buffer: ArrayBuffer): FontNameData[];
+}
+
+declare const FontName: FontName;
+export = FontName;
```

Comment:
```suggestion
export = FontName;
export as namespace FontName;
```

As the package is in UMD format so declares a global.